docs/topics/testing/tools.txt

```diff
@@ -1566,6 +1566,38 @@ your test suite.
 
     Output in case of error can be customized with the ``msg`` argument.
 
+.. method:: SimpleTestCase.assertSignalSent(signal, required_kwargs=None)
+
+    Asserts that a signal was sent. This assertion **must** be used as
+    a context manager::
+
+        with self.assertSignalSent(signals.post_save):
+            m = MyModelInstance()
```

Comment:
Seems strange to have a model class named `MyModelInstance`, even if it's a fictional one. `MyModel` might be more sensible?